XmlSchemas/ConfigurationOverrides/overrides.xsd

```diff
@@ -4,29 +4,102 @@
+                </xsd:annotation>
+              </xsd:element>
+              <xsd:element minOccurs="0" name="ServiceNounPrefix" type="xsd:string">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. 2-5 characters abbreviation used to identify cmdlets for the service. This must be unique between all the services. The only valid characters are capital letters and numbers. Services that are part of the same "family" (e.g. CloudWatch-related services) usually have similar ServiceNounPrefix. This is part of the initial configuration for the service and should't be changed after release.</xsd:documentation>
+                </xsd:annotation>
+              </xsd:element>
+              <xsd:element minOccurs="0" name="ServiceName" type="xsd:string">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. Marketing name of the service, this should be the same name used on the product page for the service. This is part of the initial configuration for the service. This can be changed after release through a PR and has mild backward compatibility concerns.</xsd:documentation>
+                </xsd:annotation>
+              </xsd:element>
+              <xsd:element minOccurs="0" name="ServiceClientInterface" type="xsd:string">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. Name of the interface for the service client (not including the namespace) in the AWS SDK for .NET. This is part of the initial configuration for the service and should't be changed after release.</xsd:documentation>
+                </xsd:annotation>
+              </xsd:element>
+              <xsd:element minOccurs="0" name="ServiceClient" type="xsd:string">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. Name of the service client class (not including the namespace) in the AWS SDK for .NET. This is part of the initial configuration for the service and should't be changed after release.</xsd:documentation>
+                </xsd:annotation>
+              </xsd:element>
+              <xsd:element minOccurs="0" name="ServiceModuleGuid" type="xsd:string">
+                <xsd:annotation>
+                  <xsd:documentation>ADVANCED CONFIGURATION. Unique UUID for the service PowerShell module. A new UUID is created manually and assigned when the module for a new service is configured. An empty ServiceModuleGuid is used if 1) the service cmdlets should be included in AWSPowerShell and AWSPowerShell.NetCore but not in AWS.Tools 2) the service cmdlets should be included in another service module (see the configuration of dynamodbstreams). This is part of the initial configuration for the service and should't be changed after release.</xsd:documentation>
```

Comment:
An empty ServiceModuleGuid is used if: 

There are other places like this also where : should be used to set off the list.

Reply:
ok